### PR TITLE
Invalid sensors: changes for Dial Box & About page

### PR DESCRIPTION
**Pull request by akampen**
This PR does a couple things:
- Fixes translations for the home and about pages for the "invalid sensor/sensor down" stuff
- Small corrections on the English versions as well
- Uses "a las" when it's not 1 o'clock, uses "a la" when it is
- If a sensor has never given a valid reading, use different text in the dial box to reflect that.

### CODE REVIEW

File: public/locales/en/dial.json
Comment (acgrobman) [inline public/locales/en/dial.json:12]: It may not be worth it, but I know i18next has built in plurals support. Is this using that?

File: public/locales/en/dial.json
Comment (acgrobman) [inline public/locales/en/dial.json:16]: Do we need a period here?

**Review by acgrobman — COMMENTED** [review]
Looks pretty good to me, haven't had ability to test myself because on mobile. Not checking off just cause waiting for Carson to come through and review as well, but consider that I've approved pending his approval

**Review by CarsonHerness — APPROVED** [review]

File: public/locales/en/dial.json
Comment (akampen) [reply public/locales/en/dial.json:16]: We don't because the beginning of the next string starts w/ a period (to account for the fact that there's no period in the time-string).

File: public/locales/en/dial.json
Comment (akampen) [reply public/locales/en/dial.json:12]: It doesn't use that. I checked it out and it would end up being a very similar implementation to what we have rn, but if it's preferred that I do that, I can switch it.

**Pull request merged**
